src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleNoSwing.kt

```diff
@@ -0,0 +1,8 @@
+package net.ccbluex.liquidbounce.features.module.modules.render
```

Comment:
Please always add the copyright header into files.

Reply:
sure, but why add copy right anyway?